edx_exams/apps/lti/tests/test_views.py

```diff
@@ -1,20 +1,248 @@
 """
 Tests for the exams LTI views
 """
+import json
 import uuid
 from unittest.mock import patch
 from urllib.parse import urljoin
 
+import ddt
+from Cryptodome.PublicKey import RSA
+from django.contrib.auth.models import AnonymousUser
 from django.urls import reverse
 from lti_consumer.data import Lti1p3LaunchData, Lti1p3ProctoringLaunchData
-from lti_consumer.models import LtiConfiguration
+from lti_consumer.lti_1p3.extensions.rest_framework.authentication import Lti1p3ApiAuthentication
+from lti_consumer.models import LtiConfiguration, LtiProctoringConsumer
 
 from edx_exams.apps.api.test_utils import ExamsAPITestCase, UserFactory
 from edx_exams.apps.core.models import CourseExamConfiguration, Exam, ExamAttempt
 from edx_exams.apps.core.statuses import ExamAttemptStatus
 from edx_exams.apps.lti.utils import get_lti_root
 
 
+@ddt.ddt
+class LtiAcsTestCase(ExamsAPITestCase):
```

Comment:
What do you think about refactoring this a little to use some of the [factories](https://github.com/edx/edx-exams/blob/main/edx_exams/apps/api/test_utils/factories.py) Zach introduced?